Add Rachel's Code

tomtclai opened the pull request:
@kunlakan 
Rachel's code compiles. 
Turns out she didn't change anything -- these are all additions

Commit:
rachel's code

Rachel-Horton commented:
Thanks Tom! Can I assume from this mail that you got the Functional API up on github? If so, you're a hero!

I did change code in the original game to support personalization. That version was saved as a library within the Functional API. 

Kelvin, if you want the edited version of Corrupted I can get that to you as soon as I get back from Texas 😀

Rachel

Sent from my iPhone

> On Mar 28, 2016, at 4:49 PM, Tom Lai notifications@github.com wrote:
> 
> Turns out she didn't change anything -- these are all additions
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/kelvinhsung/LearningGames/pull/1
> 
> Commit Summary
> 
> rachel's code
> File Changes
> 
> A CorruptedFinal/CorruptedFunctionalAPI/.classpath (8)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/CorruptedFunctionalAPI.java (705)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/LAB1.java (23)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/LAB10.java (59)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/LAB2.java (33)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/LAB3.java (35)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/LAB4.java (179)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/LAB5.java (88)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/LAB6.java (88)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/LAB7.java (107)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/LAB8.java (33)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/LAB9.java (59)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/Main.java (45)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/MethodsToOverride.java (85)
> A CorruptedFinal/CorruptedFunctionalAPI/src/Corrupted_FunctionalAPI/MyGame.java (57)
> Patch Links:
> 
> https://github.com/kelvinhsung/LearningGames/pull/1.patch
> https://github.com/kelvinhsung/LearningGames/pull/1.diff
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub

Commit:
Fixes for corrupted, fix build path error for HTL

Corrupted
- Add Rachel's Library (I imported the .jar file)
- Move labs into their own project
- Remove unused imports to silence eclipse warnings
- Remove buildpath link to the old API

HTL
- Remove non-existent buildpath to compile the project

tomtclai commented:
@kunlakan 
This last commit https://github.com/kelvinhsung/LearningGames/pull/1/commits/40d4b23fd6a995dedb61f974c43647d212ad7271: 

Corrupted
- Add Rachel's Library (I _imported_ the `CorruptedLibraryFiles.jar` file-- I'm not sure if that's the intended usage)
- Move labs into their own project
- Remove unused imports to silence eclipse warnings
- Remove buildpath links to the old API `Corrupted` (except in `CorruptedUnit` -- we are not sure about `CorruptedUnit`)
